### PR TITLE
Feature/rewrite code

### PR DESCRIPTION
**Pull request by dunamu-keira**
변경 내용: 
* 트레이닝 변수값 content 에서 시작
* vgg16에서 vgg19로 변경
* mean pixel값으로 각 이미지들을 normalized 함

### CODE REVIEW

File: neural-style/neural-style.py [hunk 1]
```diff
@@ -1,86 +1,78 @@
...
-    reshaped_variable = tf.reshape(variable_layer, [-1, N])
-    G = tf.matmul(reshaped_variable, reshaped_variable, transpose_a=True, transpose_b=False)
-    reshaped_style = tf.reshape(style_layer, [-1, N])
-    A = tf.matmul(reshaped_style, reshaped_style, transpose_a=True, transpose_b=False)
-    return tf.nn.l2_loss(G - A) / (2 * N**2 * M ** 2)
+    reshaped_variable = tf.reshape(variable_layer, (-1, N))
+    G = tf.matmul(reshaped_variable, reshaped_variable, transpose_a=True)
+    reshaped_style = tf.reshape(style_layer, (-1, N))
+    A = tf.matmul(reshaped_style, reshaped_style, transpose_a=True)
+    return tf.nn.l2_loss(G - A) / (4.0 * N ** 2 * M ** 2)
```
Comment (lyuha) [inline neural-style/neural-style.py]: 4.0 -> 2.0 이 원래 식에 더 가까울 거에요.
l2_loss가 output = sum(t ** 2) / 2 식이라서

Comment (dunamu-keira) [reply neural-style/neural-style.py]: 이 식 베낀거임ㅋㅋㅋ 그리고 큰 영향을 줄거같진 않은데

Comment (lyuha) [reply neural-style/neural-style.py]: 어차피 style_loss에 /2 니까 콘텐츠 로스에 비해서 저평가 될 수 있다는 점? 정도 있겠네요. alpha / beta 비율 생각하면 의미 없을 수도 있네요

Comment (dunamu-keira) [reply neural-style/neural-style.py]: 네ㅋㅋ 이미 alpha : beta 비율도 논문과 다르게 1 : 10000 이므로 결과적으로 보면 1 : 20000이 되는 정도.. 근데 깔끔한 수식 + 논문의 순수성(???)을 원한다면 2.0으로 바꿔도 상관없음ㅋㅋ
